go.mod

```diff
@@ -75,6 +75,7 @@ require (
 	go.etcd.io/etcd/client/v3 v3.5.0
 	go.etcd.io/etcd/server/v3 v3.5.0
 	go.etcd.io/etcd/tests/v3 v3.5.0
+	go.uber.org/atomic v1.7.0
```

Comment:
:thinking: not sure where this dependency comes from, does it persist if you run `go mod tidy`?